src/message.rs

```diff
@@ -31,7 +30,7 @@ fn msg_send_fn<R: Any>() -> unsafe extern fn(*mut Object, Sel, ...) -> R {
     }
 }
 
-#[cfg(target_arch = "x86")]
+#[cfg(all(target_arch = "x86", not(feature = "gnustep_runtime")))]
 fn msg_send_super_fn<R: Any>() -> unsafe extern fn(*mut Object, Sel, ...) -> R {
```

Comment:
Why does this have to be `cfg`'d out? It didn't look like any runtime methods were removed in gnustep, so I'd expect it to still compile.